README.md

```diff
@@ -1,5 +1,117 @@
+## Problem Statement
+On the modern web, we frequently access the same content on a variety of
+platforms. Each device will display content differently due to their sizes
+and proportions, causing text to shrink, or parts of the page to shift around,
+and can lead to poor user experience. As front end developers, we want our 
+web sites to look good and function well regardless of whether a user is on 
+their laptop, phone or tablet. This has been an ongoing challenge, so what 
+strategies have been developed to address these issues?
+
+## Objectives
+
+1. Using server-side detection for redirection
+2. Creating responsive layouts with CSS
+3. What is desktop-down design
+3. What is mobile-up design
+4. Explain graceful degradation
+5. Explain progressive enhancement
+6. Why make users update their browsers
+
+## Using Server-Side Detection for Redirection
+
+It is possible for web servers to check the requesting device's headers info,
+and send back files specific to the type of device. Going to
+[twitter.com](twitter.com) on your phone will automatically redirect you to
+[mobile.twitter.com](mobile.twitter.com) because their servers detected that a
+phone was sending the request.
+
+Unfortunately, while this is a common solution, it requires building _two_
+separate websites. Every time you want to change something about the content
+of your site, you have to do it twice, once for both the desktop and mobile
+versions of your site.
+
+## Creating Responsive Layouts with CSS
+
+The other common solution to is to use Cascading Style Sheets.  Instead of a
+server detecting the type of device, every user receives the same content,
+and the provided CSS informs the user's browser on how to style the DOM. A well
+designed, responsive website relies on a combination of CSS media queries,
+layout properties like flexbox and grid, and HTML viewport settings.
+
+## What is Desktop-Down Design?
+
+There are multiple approaches to responsive design.  One approach is
+_Desktop-down_, where a site is first designed for desktop devices, then
+modified layouts are developed for tablet and mobile devices. Since we're
+_doing_ the work of designing a website on a desktop or laptop already, this
+approach can feel the most natural.  Desktops and laptops offer more
+capabilities with their more powerful hardware, and the larger screen size means
+a larger space to work with, a larger canvas for us to paint on.
+
+Desktop-down design often means that, by default, elements on the page are the
+width and height they should be for a larger screen.  When a smaller screen size
+is detected, the default styling will be overridden and replaced with the rules
+for tablet or mobile styling
+
+## What is Mobile-Up Design
+
+Going in the opposite direction is _Mobile-up_, an approach gaining popularity in
```

Comment:
1. Downcase Mobile here, Desktop in the above section. They're not proper nouns.

2. Add "...or _mobile-first_"